google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -78,6 +78,7 @@
   private static final String REFRESH_TOKEN_LABEL = "refresh_token";
   private static final String GCLOUD_USER_TYPE_LABEL = "type";
   private static final String GCLOUD_USER_TYPE = "authorized_user";
+  private static final String DOCKERFILE = "/Dockerfile";
```

Comment:
I'd be tempted to set this to 'Dockerfile', since '/Dockerfile' alone is a valid path to the root of the file system, which will never be what we want.